python/llm/test/benchmark/perplexity/ppl_csv_to_html.py

```diff
@@ -150,40 +167,48 @@ def main():
+                        diff_ppl_result[latest_csv_ind]=round((latest_ppl_result-previous_ppl_result)*100/previous_ppl_result,2)
+                        # last_zh[latest_csv_ind]=previous_zh
+                        # diff_zh[latest_csv_ind]=round((latest_zh-previous_zh)*100/previous_zh,2)
                         in_previous_flag=True
 
+                        # last_en[latest_csv_ind]=previous_en
+                        # diff_en[latest_csv_ind]=round((latest_en-previous_en)*100/previous_en,2)
+                        # last_zh[latest_csv_ind]=previous_zh
+                        # diff_zh[latest_csv_ind]=round((latest_zh-previous_zh)*100/previous_zh,2)
+                        # in_previous_flag=True
+
             if not in_previous_flag:
-                last_en[latest_csv_ind]=pd.NA
-                diff_en[latest_csv_ind]=pd.NA
-                last_zh[latest_csv_ind]=pd.NA
-                diff_zh[latest_csv_ind]=pd.NA
+                last_ppl_result[latest_csv_ind]=pd.NA
+                diff_ppl_result[latest_csv_ind]=pd.NA
+                # last_zh[latest_csv_ind]=pd.NA
+                # diff_zh[latest_csv_ind]=pd.NA
 
-        latest_csv.insert(loc=9,column='last_en',value=last_en)
-        latest_csv.insert(loc=10,column='diff_en(%)',value=diff_en)
-        latest_csv.insert(loc=11,column='last_zh',value=last_zh)
-        latest_csv.insert(loc=12,column='diff_zh(%)',value=diff_zh)
+        latest_csv.insert(loc=6,column='last_ppl_result',value=last_ppl_result)
+        latest_csv.insert(loc=7,column='last_diff_ppl_result(%)',value=diff_ppl_result)
+        # latest_csv.insert(loc=11,column='last_zh',value=last_zh)
+        # latest_csv.insert(loc=12,column='diff_zh(%)',value=diff_zh)
 
 
-        diffs_within_normal_range = is_diffs_within_normal_range(diff_en, diff_zh, threshold=highlight_threshold)
+        diffs_within_normal_range = is_diffs_within_normal_range(diff_ppl_result, threshold=highlight_threshold)#en, diff_zh, threshold=highlight_threshold)
 
-        subset1=['diff_en(%)','diff_zh(%)']
+        subset1=['last_diff_ppl_result(%)']#['diff_en(%)','diff_zh(%)']
         
-        columns={'en': '{:.2f}', 'zh': '{:.2f}', 'last_en': '{:.2f}', 'diff_en(%)': '{:.2f}',
-                'last_zh': '{:.2f}', 'diff_zh(%)': '{:.2f}'}
-
+        columns={'ppl_result': '{:.2f}', 'last_ppl_result': '{:.2f}', 'last_diff_ppl_result(%)': '{:.2f}'}
+                #{'en': '{:.2f}', 'zh': '{:.2f}', 'last_en': '{:.2f}', 'diff_en(%)': '{:.2f}',
+                #'last_zh': '{:.2f}', 'diff_zh(%)': '{:.2f}'}
         latest_csv.drop('Index', axis=1, inplace=True)
 
         styled_df = latest_csv.style.format(columns).applymap(lambda val: highlight_vals(val, max=3.0, is_last=True), subset=subset1)
-        for task in ['en', 'zh']:
+        for task in ['ppl_result']:#['en', 'zh']:
             styled_df = styled_df.applymap(lambda val: highlight_vals(val, max=highlight_threshold, is_last=False), subset=[f'{task}_diff_FP16(%)'])
```

Comment:
delete `for task in ['ppl_result']:#['en', 'zh']:` and change `task` to `'ppl_result'`